docs/api/Rectangle.md

```diff
@@ -54,18 +54,31 @@ var newRect = rect.changeBasis({
   from: layerA,
   to: layerB,
 })
+
+var parentRect = rect.changeBasis({
+  from: layerA,
+  to: layerA.parent,
+})
+
+var pageRect = rect.changeBasis({
+  from: layerA,
+  // leaving out `to` means changing the
+  // basis to the Page's basis
+})
 ```
 
-Each layer defines its own coordinates basis. You can change that basis from one layer to the other with `changeBasis`.
+Each layer defines its own system of coordinates (with its origin at the top left of the layer). You can change that [basis](<https://en.wikipedia.org/wiki/Basis_(linear_algebra)>) from one layer to the other with `changeBasis`.
 
-| Parameters                                            |                                                                   |
-| ----------------------------------------------------- | ----------------------------------------------------------------- |
-| change<span class="arg-type">object - required</span> |                                                                   |
-| change.from<span class="arg-type">Layer</span>        | The layer in which the rectangle's coordinates are expressed.     |
-| change.to<span class="arg-type">Layer</span>          | The layer in which the rectangle's coordinates will be expressed. |
+| Parameters                                               |                                                                   |
+| -------------------------------------------------------- | ----------------------------------------------------------------- |
+| change<span class="arg-type">object - required</span>    |                                                                   |
+| change.from<span class="arg-type">[Layer](#layer)</span> | The layer in which the rectangle's coordinates are expressed.     |
+| change.to<span class="arg-type">[Layer](#layer)</span>   | The layer in which the rectangle's coordinates will be expressed. |
 
 Both `from` and `to` can be ommited (but not at the same time) to change the basis from/to the Page coordinates.
 
+If you want to explore the beautiful world of linear algebra from which the term `Basis` originates from, Khan Academy has some really nice [videos](https://www.khanacademy.org/math/linear-algebra/alternate-bases/change-of-basis/v/linear-algebra-change-of-basis-matrix).
```

Comment:
heh I think the top description is fine - this is a bit superfluous and just sort of pollutes the API

Reply:
hahaha

Reply:
🙈